### ISSUE TITLE
Evolution Acteur Metier Grapheur

### ISSUE THREAD

**Issue by WITNicolas** [witsa/synapps#72]
Pouvoir enlever la barre de navigation.
Avoir la propriété de navigation éditable dans l'objet. Cette propriété pouvant être asservi par la suite à d'autres process.
Cela permet de faire 2 objets superposés avec une opacité de 50% avec des périodes d'affichage différents. Par exemple les consos mensuelles de 2018 superposées aux consos mensuelles de 2017.
Ou bien des historiques de températures sur des périodes différentes également.

Très utile pour des affichages de sensibilisation sur écran. Il faut toujours faire beaucoup de comparaison Avant/Après pour sensibiliser les gens sur l'impact de leur comportement.

**Comment by ericvoignier** [comment]
Oui concernant la propriété de navigation, l'acteur grapheur va évoluer pour être plus customisable.
Concernant, la comparaison sur des périodes différentes, c'est effectivement indispensable. Ta solution est un moyen de contournement intéressant, même si il  me semble que c'est le grapheur lui-même qui devrait rendre cela possible dans la préparation. @manufitoussiwit est-ce possible a intégrer dans la roadmap du grapheur ?

**Comment by WITNicolas** [comment]
En fait le charts qui est dans le tableau de bord et bien adapté à cela pour base de fonctionnement
![image](https://user-images.githubusercontent.com/33632664/35819319-a3a8a61c-0aa2-11e8-844a-cad50f64cf7a.png)
Sauf qu'au lieu d'avoir plusieurs séries côte à côte, il faudrait la même série sur des périodes différentes, sélectionnables.
Exemple en rouge 2018 / Orange 2017
Bref vous avez bien compris le truc je pense.

Personnellement ça ne me gêne pas que cela ne soit pas directement lié aux FLUX plutôt qu'aux TRACES.
Le tableau de bord fonctionne uniquement avec les FLUX et ce n'est pas gênant, de plus toute la mécanique est déjà faite dans une certaine mesure.

**Comment by ericvoignier** [comment]
Oui, ma proposition concernait le grapheur, mais effectivement cela est pertinent sur le charting WIT. L'étape préalable est dans un premier temps d'intégrer le charting en tant qu'acteur SynApps !
